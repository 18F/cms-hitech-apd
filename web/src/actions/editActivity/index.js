export * from './contractorResources';
export * from './costAllocate';
export * from './overview';
