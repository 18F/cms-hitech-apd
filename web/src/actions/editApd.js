export const EDIT_APD = Symbol('edit apd');

export const setMedicaidDirectorEmail = email => {
  return {
    type: EDIT_APD,
    path: '/stateProfile/medicaidDirector/email',
    value: email
  };
};

export const setMedicaidDirectorName = name => {
  return {
    type: EDIT_APD,
    path: '/stateProfile/medicaidDirector/name',
    value: name
  };
};

export const setMedicaidDirectorPhoneNumber = phone => {
  return {
    type: EDIT_APD,
    path: '/stateProfile/medicaidDirector/phone',
    value: phone
  };
};

export const setMedicaidOfficeAddress1 = address => {
  return {
    type: EDIT_APD,
    path: '/stateProfile/medicaidOffice/address1',
    value: address
  };
};

export const setMedicaidOfficeAddress2 = address => {
  return {
    type: EDIT_APD,
    path: '/stateProfile/medicaidOffice/address2',
    value: address
  };
};

export const setMedicaidOfficeCity = city => {
  return {
    type: EDIT_APD,
    path: '/stateProfile/medicaidOffice/city',
    value: city
  };
};

export const setMedicaidOfficeState = state => {
  return {
    type: EDIT_APD,
    path: '/stateProfile/medicaidOffice/state',
    value: state
  };
};

export const setMedicaidOfficeZip = zip => {
  return {
    type: EDIT_APD,
    path: '/stateProfile/medicaidOffice/zip',
    value: zip
  };
};
