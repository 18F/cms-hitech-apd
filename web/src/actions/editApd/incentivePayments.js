import { EDIT_APD } from './symbols';

export const setIncentiveEHCount = (year, quarter, value) => ({
  type: EDIT_APD,
  path: `/incentivePayments/ehCt/${year}/${quarter}`,
  value
});

export const setIncentiveEHPayment = (year, quarter, value) => ({
  type: EDIT_APD,
  path: `/incentivePayments/ehAmt/${year}/${quarter}`,
  value
});

export const setIncentiveEPCount = (year, quarter, value) => ({
  type: EDIT_APD,
  path: `/incentivePayments/epCt/${year}/${quarter}`,
  value
});

export const setIncentiveEPPayment = (year, quarter, value) => ({
  type: EDIT_APD,
  path: `/incentivePayments/epAmt/${year}/${quarter}`,
  value
});
