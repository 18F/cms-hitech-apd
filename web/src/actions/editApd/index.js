export * from './symbols';

export * from './keyPersonnel';
export * from './overview';
export * from './previousActivities';
export * from './stateProfile';
