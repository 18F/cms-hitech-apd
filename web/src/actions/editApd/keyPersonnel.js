import { ADD_APD_ITEM, EDIT_APD, REMOVE_APD_ITEM } from './symbols';
import { updateBudget } from '../apd';

export const addKeyPerson = () => (dispatch, getState) => {
  dispatch({
    type: ADD_APD_ITEM,
    path: '/keyPersonnel/-',
    state: getState()
  });
};

export const removeKeyPerson = (
  index,
  { global = window } = {}
) => dispatch => {
  if (global.confirm('Do you really want to delete this key person?')) {
    dispatch({
      type: REMOVE_APD_ITEM,
      path: `/keyPersonnel/${index}`
    });
    dispatch(updateBudget());
  }
};

export const setKeyPersonName = (name, index) => ({
  type: EDIT_APD,
  path: `/keyPersonnel/${index}/name`,
  value: name
});

export const setKeyPersonEmail = (email, index) => ({
  type: EDIT_APD,
  path: `/keyPersonnel/${index}/email`,
  value: email
});

export const setKeyPersonRole = (role, index) => ({
  type: EDIT_APD,
  path: `/keyPersonnel/${index}/position`,
  value: role
});

export const setKeyPersonPercentTime = (percent, index) => ({
  type: EDIT_APD,
  path: `/keyPersonnel/${index}/percentTime`,
  value: percent
});

// needs to trigger budget calculation
export const setKeyPersonHasCosts = (hasCosts, index) => dispatch => {
  dispatch({
    type: EDIT_APD,
    path: `/keyPersonnel/${index}/hasCosts`,
    value: hasCosts
  });
  dispatch(updateBudget());
};

// needs to trigger budget calculation
export const setKeyPersonCost = (cost, index, year) => dispatch => {
  dispatch({
    type: EDIT_APD,
    path: `/keyPersonnel/${index}/costs/${year}`,
    value: cost
  });
  dispatch(updateBudget());
};
