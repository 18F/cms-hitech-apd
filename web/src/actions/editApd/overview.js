import { ADD_APD_YEAR, EDIT_APD, REMOVE_APD_YEAR } from './symbols';
import { updateBudget } from '../apd';

export const addYear = year => (dispatch, getState) => {
  dispatch({ type: ADD_APD_YEAR, value: year, state: getState() });
  dispatch(updateBudget());
};

export const removeYear = year => (dispatch, getState) => {
  dispatch({ type: REMOVE_APD_YEAR, value: year, state: getState() });
  dispatch(updateBudget());
};

export const setNarrativeForHIE = text => ({
  type: EDIT_APD,
  path: '/narrativeHIE',
  value: text
});

export const setNarrativeForHIT = text => ({
  type: EDIT_APD,
  path: '/narrativeHIT',
  value: text
});

export const setNarrativeForMMIS = text => ({
  type: EDIT_APD,
  path: '/narrativeMMIS',
  value: text
});

export const setProgramOverview = text => ({
  type: EDIT_APD,
  path: '/programOverview',
  value: text
});
