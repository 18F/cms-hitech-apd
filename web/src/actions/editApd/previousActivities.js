import { EDIT_APD } from './symbols';

export const setPreviousActivitySummary = summary => ({
  type: EDIT_APD,
  path: '/previousActivitySummary',
  value: summary
});

export const setPreviousActivityApprovedExpenseForHITandHIE = (
  year,
  expense
) => ({
  type: EDIT_APD,
  path: `/previousActivityExpenses/${year}/hithie/totalApproved`,
  value: expense
});

export const setPreviousActivityFederalActualExpenseForHITandHIE = (
  year,
  expense
) => ({
  type: EDIT_APD,
  path: `/previousActivityExpenses/${year}/hithie/federalActual`,
  value: expense
});

export const setPreviousActivityApprovedExpenseforMMIS50FFP = (
  year,
  expense
) => ({
  type: EDIT_APD,
  path: `/previousActivityExpenses/${year}/mmis/50/totalApproved`,
  value: expense
});

export const setPreviousActivityApprovedExpenseforMMIS75FFP = (
  year,
  expense
) => ({
  type: EDIT_APD,
  path: `/previousActivityExpenses/${year}/mmis/75/totalApproved`,
  value: expense
});

export const setPreviousActivityApprovedExpenseforMMIS90FFP = (
  year,
  expense
) => ({
  type: EDIT_APD,
  path: `/previousActivityExpenses/${year}/mmis/90/totalApproved`,
  value: expense
});

export const setPreviousActivityFederalActualExpenseforMMIS50FFP = (
  year,
  expense
) => ({
  type: EDIT_APD,
  path: `/previousActivityExpenses/${year}/mmis/50/federalActual`,
  value: expense
});

export const setPreviousActivityFederalActualExpenseforMMIS75FFP = (
  year,
  expense
) => ({
  type: EDIT_APD,
  path: `/previousActivityExpenses/${year}/mmis/75/federalActual`,
  value: expense
});

export const setPreviousActivityFederalActualExpenseforMMIS90FFP = (
  year,
  expense
) => ({
  type: EDIT_APD,
  path: `/previousActivityExpenses/${year}/mmis/90/federalActual`,
  value: expense
});
