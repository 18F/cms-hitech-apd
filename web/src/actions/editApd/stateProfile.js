import { EDIT_APD } from './symbols';

export const setMedicaidDirectorEmail = email => ({
  type: EDIT_APD,
  path: '/stateProfile/medicaidDirector/email',
  value: email
});

export const setMedicaidDirectorName = name => ({
  type: EDIT_APD,
  path: '/stateProfile/medicaidDirector/name',
  value: name
});

export const setMedicaidDirectorPhoneNumber = phone => ({
  type: EDIT_APD,
  path: '/stateProfile/medicaidDirector/phone',
  value: phone
});

export const setMedicaidOfficeAddress1 = address => ({
  type: EDIT_APD,
  path: '/stateProfile/medicaidOffice/address1',
  value: address
});

export const setMedicaidOfficeAddress2 = address => ({
  type: EDIT_APD,
  path: '/stateProfile/medicaidOffice/address2',
  value: address
});

export const setMedicaidOfficeCity = city => ({
  type: EDIT_APD,
  path: '/stateProfile/medicaidOffice/city',
  value: city
});

export const setMedicaidOfficeState = state => ({
  type: EDIT_APD,
  path: '/stateProfile/medicaidOffice/state',
  value: state
});

export const setMedicaidOfficeZip = zip => {
  return {
    type: EDIT_APD,
    path: '/stateProfile/medicaidOffice/zip',
    value: zip
  };
};
