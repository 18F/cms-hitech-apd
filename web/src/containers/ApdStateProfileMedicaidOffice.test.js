import { mount } from 'enzyme';
import React from 'react';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk';

import { EDIT_APD } from '../actions/editApd';

import StateProfile from './ApdStateProfileMedicaidOffice';

const mockStore = configureStore([thunk]);

describe('apd state profile, Medicaid office component', () => {
  const store = mockStore({
    apd: {
      data: {
        stateProfile: {
          medicaidDirector: {
            name: 'name',
            email: 'email',
            phone: 'phone'
          },
          medicaidOffice: {
            address1: 'address 1',
            address2: 'address 2',
            city: 'city',
            state: 'state',
            zip: 'zip'
          }
        }
      }
    },
    user: {
      data: {
        state: { id: 'mn' }
      }
    }
  });

  beforeEach(() => {
    store.clearActions();
  });

  test('renders correctly', () => {
    expect(
      mount(
        <Provider store={store}>
          <StateProfile />
        </Provider>
      )
    ).toMatchSnapshot();
  });

  [
    {
      testName: 'director name change',
      fieldName: 'apd-state-profile-mdname',
      path: '/stateProfile/medicaidDirector/name'
    },
    {
      testName: 'director email change',
      fieldName: 'apd-state-profile-mdemail',
      path: '/stateProfile/medicaidDirector/email'
    },
    {
      testName: 'director phone number change',
      fieldName: 'apd-state-profile-mdphone',
      path: '/stateProfile/medicaidDirector/phone'
    },
    {
      testName: 'office address line 1 change',
      fieldName: 'apd-state-profile-addr1',
      path: '/stateProfile/medicaidOffice/address1'
    },
    {
      testName: 'office address line 2 change',
      fieldName: 'apd-state-profile-addr2',
      path: '/stateProfile/medicaidOffice/address2'
    },
    {
      testName: 'office city change',
      fieldName: 'apd-state-profile-city',
      path: '/stateProfile/medicaidOffice/city'
    },
    {
      testName: 'office state change',
      fieldName: 'apd-state-profile-state',
      path: '/stateProfile/medicaidOffice/state'
    },
    {
      testName: 'office zip change',
      fieldName: 'apd-state-profile-zip',
      path: '/stateProfile/medicaidOffice/zip'
    }
  ].forEach(({ testName, fieldName, path }) => {
    test(`dispatches on ${testName}`, () => {
      const component = mount(
        <Provider store={store}>
          <StateProfile />
        </Provider>
      );

      component
        .findWhere(c => c.prop('name') === fieldName)
        .first()
        .prop('onChange')({ target: { value: 'new text' } });

      const expectedActions = [
        {
          type: EDIT_APD,
          path,
          value: 'new text'
        }
      ];

      expect(store.getActions()).toEqual(expectedActions);
    });
  });
});
