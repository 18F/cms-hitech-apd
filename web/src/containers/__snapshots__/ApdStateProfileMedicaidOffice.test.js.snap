// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`apd state profile, Medicaid office component renders correctly 1`] = `
<Provider
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <ApdStateProfile>
    <fieldset>
      <legend
        className="ds-u-padding-bottom--1"
      >
        Medicaid director
      </legend>
      <TextField
        label="Name"
        name="apd-state-profile-mdname"
        onChange={[Function]}
        type="text"
        value="name"
      >
        <div
          className="ds-u-clearfix"
        >
          <FormLabel
            component="label"
            fieldId="textfield_1"
          >
            <label
              className="ds-c-label"
              htmlFor="textfield_1"
            >
              <span
                className=""
              >
                Name
              </span>
            </label>
          </FormLabel>
          <input
            className="ds-c-field"
            id="textfield_1"
            name="apd-state-profile-mdname"
            onChange={[Function]}
            type="text"
            value="name"
          />
        </div>
      </TextField>
      <TextField
        label="Email address"
        name="apd-state-profile-mdemail"
        onChange={[Function]}
        type="text"
        value="email"
      >
        <div
          className="ds-u-clearfix"
        >
          <FormLabel
            component="label"
            fieldId="textfield_2"
          >
            <label
              className="ds-c-label"
              htmlFor="textfield_2"
            >
              <span
                className=""
              >
                Email address
              </span>
            </label>
          </FormLabel>
          <input
            className="ds-c-field"
            id="textfield_2"
            name="apd-state-profile-mdemail"
            onChange={[Function]}
            type="text"
            value="email"
          />
        </div>
      </TextField>
      <TextField
        label="Phone number"
        name="apd-state-profile-mdphone"
        onChange={[Function]}
        type="text"
        value="phone"
      >
        <div
          className="ds-u-clearfix"
        >
          <FormLabel
            component="label"
            fieldId="textfield_3"
          >
            <label
              className="ds-c-label"
              htmlFor="textfield_3"
            >
              <span
                className=""
              >
                Phone number
              </span>
            </label>
          </FormLabel>
          <input
            className="ds-c-field"
            id="textfield_3"
            name="apd-state-profile-mdphone"
            onChange={[Function]}
            type="text"
            value="phone"
          />
        </div>
      </TextField>
    </fieldset>
    <fieldset>
      <legend
        className="ds-u-padding-bottom--1"
      >
        Medicaid office
      </legend>
      <TextField
        label="Address"
        name="apd-state-profile-addr1"
        onChange={[Function]}
        type="text"
        value="address 1"
      >
        <div
          className="ds-u-clearfix"
        >
          <FormLabel
            component="label"
            fieldId="textfield_4"
          >
            <label
              className="ds-c-label"
              htmlFor="textfield_4"
            >
              <span
                className=""
              >
                Address
              </span>
            </label>
          </FormLabel>
          <input
            className="ds-c-field"
            id="textfield_4"
            name="apd-state-profile-addr1"
            onChange={[Function]}
            type="text"
            value="address 1"
          />
        </div>
      </TextField>
      <TextField
        hint="Optional"
        label="Address continued"
        name="apd-state-profile-addr2"
        onChange={[Function]}
        type="text"
        value="address 2"
      >
        <div
          className="ds-u-clearfix"
        >
          <FormLabel
            component="label"
            fieldId="textfield_5"
            hint="Optional"
          >
            <label
              className="ds-c-label"
              htmlFor="textfield_5"
            >
              <span
                className=""
              >
                Address continued
              </span>
              <span
                className="ds-c-field__hint"
              >
                Optional
              </span>
            </label>
          </FormLabel>
          <input
            className="ds-c-field"
            id="textfield_5"
            name="apd-state-profile-addr2"
            onChange={[Function]}
            type="text"
            value="address 2"
          />
        </div>
      </TextField>
      <div
        className="ds-l-row"
      >
        <TextField
          className="ds-l-col--6"
          label="City"
          name="apd-state-profile-city"
          onChange={[Function]}
          type="text"
          value="city"
        >
          <div
            className="ds-u-clearfix ds-l-col--6"
          >
            <FormLabel
              component="label"
              fieldId="textfield_6"
            >
              <label
                className="ds-c-label"
                htmlFor="textfield_6"
              >
                <span
                  className=""
                >
                  City
                </span>
              </label>
            </FormLabel>
            <input
              className="ds-c-field"
              id="textfield_6"
              name="apd-state-profile-city"
              onChange={[Function]}
              type="text"
              value="city"
            />
          </div>
        </TextField>
        <div
          className="ds-u-clearfix ds-l-col--6"
        >
          <FormLabel
            component="label"
          >
            <label
              className="ds-c-label"
            >
              <span
                className=""
              >
                State
              </span>
            </label>
          </FormLabel>
          <Select
            label="State"
            name="apd-state-profile-state"
            onChange={[Function]}
            value="state"
          >
            <select
              className="ds-c-field"
              id="select_apd-state-profile-state_9"
              label="State"
              name="apd-state-profile-state"
              onChange={[Function]}
              value="state"
            >
              <option
                key="Alabama"
                value="AL"
              >
                Alabama
              </option>
              <option
                key="Alaska"
                value="AK"
              >
                Alaska
              </option>
              <option
                key="Arizona"
                value="AZ"
              >
                Arizona
              </option>
              <option
                key="Arkansas"
                value="AR"
              >
                Arkansas
              </option>
              <option
                key="California"
                value="CA"
              >
                California
              </option>
              <option
                key="Colorado"
                value="CO"
              >
                Colorado
              </option>
              <option
                key="Connecticut"
                value="CT"
              >
                Connecticut
              </option>
              <option
                key="Delaware"
                value="DE"
              >
                Delaware
              </option>
              <option
                key="Florida"
                value="FL"
              >
                Florida
              </option>
              <option
                key="Georgia"
                value="GA"
              >
                Georgia
              </option>
              <option
                key="Hawaii"
                value="HI"
              >
                Hawaii
              </option>
              <option
                key="Idaho"
                value="ID"
              >
                Idaho
              </option>
              <option
                key="Iowa"
                value="IA"
              >
                Iowa
              </option>
              <option
                key="Illinois"
                value="IL"
              >
                Illinois
              </option>
              <option
                key="Indiana"
                value="IN"
              >
                Indiana
              </option>
              <option
                key="Kansas"
                value="KS"
              >
                Kansas
              </option>
              <option
                key="Kentucky"
                value="KY"
              >
                Kentucky
              </option>
              <option
                key="Louisiana"
                value="LA"
              >
                Louisiana
              </option>
              <option
                key="Maine"
                value="ME"
              >
                Maine
              </option>
              <option
                key="Maryland"
                value="MD"
              >
                Maryland
              </option>
              <option
                key="Massachusetts"
                value="MA"
              >
                Massachusetts
              </option>
              <option
                key="Michigan"
                value="MI"
              >
                Michigan
              </option>
              <option
                key="Minnesota"
                value="MN"
              >
                Minnesota
              </option>
              <option
                key="Mississippi"
                value="MS"
              >
                Mississippi
              </option>
              <option
                key="Missouri"
                value="MO"
              >
                Missouri
              </option>
              <option
                key="Montana"
                value="MT"
              >
                Montana
              </option>
              <option
                key="Nebraska"
                value="NE"
              >
                Nebraska
              </option>
              <option
                key="North Carolina"
                value="NC"
              >
                North Carolina
              </option>
              <option
                key="North Dakota"
                value="ND"
              >
                North Dakota
              </option>
              <option
                key="New Hampshire"
                value="NH"
              >
                New Hampshire
              </option>
              <option
                key="New Jersey"
                value="NJ"
              >
                New Jersey
              </option>
              <option
                key="New Mexico"
                value="NM"
              >
                New Mexico
              </option>
              <option
                key="Nevada"
                value="NV"
              >
                Nevada
              </option>
              <option
                key="New York"
                value="NY"
              >
                New York
              </option>
              <option
                key="Ohio"
                value="OH"
              >
                Ohio
              </option>
              <option
                key="Oklahoma"
                value="OK"
              >
                Oklahoma
              </option>
              <option
                key="Oregon"
                value="OR"
              >
                Oregon
              </option>
              <option
                key="Pennsylvania"
                value="PA"
              >
                Pennsylvania
              </option>
              <option
                key="Rhode Island"
                value="RI"
              >
                Rhode Island
              </option>
              <option
                key="South Carolina"
                value="SC"
              >
                South Carolina
              </option>
              <option
                key="South Dakota"
                value="SD"
              >
                South Dakota
              </option>
              <option
                key="Tennessee"
                value="TN"
              >
                Tennessee
              </option>
              <option
                key="Texas"
                value="TX"
              >
                Texas
              </option>
              <option
                key="Utah"
                value="UT"
              >
                Utah
              </option>
              <option
                key="Vermont"
                value="VT"
              >
                Vermont
              </option>
              <option
                key="Virginia"
                value="VA"
              >
                Virginia
              </option>
              <option
                key="Washington"
                value="WA"
              >
                Washington
              </option>
              <option
                key="West Virginia"
                value="WV"
              >
                West Virginia
              </option>
              <option
                key="Wisconsin"
                value="WI"
              >
                Wisconsin
              </option>
              <option
                key="Wyoming"
                value="WY"
              >
                Wyoming
              </option>
              <option
                key="District of Columbia"
                value="DC"
              >
                District of Columbia
              </option>
              <option
                key="American Samoa"
                value="AS"
              >
                American Samoa
              </option>
              <option
                key="Guam"
                value="GU"
              >
                Guam
              </option>
              <option
                key="Northern Mariana Islands"
                value="MP"
              >
                Northern Mariana Islands
              </option>
              <option
                key="Puerto Rico"
                value="PR"
              >
                Puerto Rico
              </option>
              <option
                key="U.S. Virgin Islands"
                value="VI"
              >
                U.S. Virgin Islands
              </option>
            </select>
          </Select>
        </div>
      </div>
      <TextField
        label="ZIP Code"
        mask="zip"
        name="apd-state-profile-zip"
        onChange={[Function]}
        type="text"
        value="zip"
      >
        <div
          className="ds-u-clearfix"
        >
          <FormLabel
            component="label"
            fieldId="textfield_8"
          >
            <label
              className="ds-c-label"
              htmlFor="textfield_8"
            >
              <span
                className=""
              >
                ZIP Code
              </span>
            </label>
          </FormLabel>
          <div
            className="ds-c-field-mask ds-c-field-mask--zip"
          >
            <div
              className="ds-c-field__before ds-c-field__before--zip"
            />
            <Mask
              mask="zip"
            >
              <input
                className="ds-c-field ds-c-field--zip"
                id="textfield_8"
                name="apd-state-profile-zip"
                onBlur={[Function]}
                onChange={[Function]}
                type="text"
                value="zip"
              />
            </Mask>
          </div>
        </div>
      </TextField>
    </fieldset>
  </ApdStateProfile>
</Provider>
`;
